README.md

```diff
@@ -227,10 +227,27 @@ result in a test reporting a false positive error.
 
 ## Contributing
 
-Ensure to run `mypy` and `black` (found in
-[requirements-dev.txt](requirements-dev.txt)) during development to ensure
-smooth pull requests.
+Before submitting a pull request, run the following error-checking and formatting tools (found in
+[requirements-dev.txt](requirements-dev.txt)):
+
+* [mypy][3]: `mypy src`
+  * mypy checks for typing errors. You should resolve all typing errors or if an error cannot be 
+    resolved (e.g., it's due to a third-party library), you should add a comment 
+    `# type: ignore` to [silence][4] the error.
+* [docformatter][5]: `docformatter -i src`
+  * This formats docstrings. You should review and add any changes to your PR.
+* [Black][6]: `black src`
```

Comment:
We should run tooling on both src and tests directories.
`mypy src` -> `mypy src tests` (same for others)

Reply:
Whoops, fixed.